anghammarad/src/test/scala/com/gu/anghammarad/serialization/SerializationTest.scala

```diff
@@ -5,18 +5,20 @@ import com.amazonaws.services.lambda.runtime.events.SNSEvent.SNSRecord
 import com.gu.anghammarad.models._
 import io.circe.Json
 import io.circe.parser._
-import org.scalatest.{EitherValues, FreeSpec, Matchers}
-
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
 import com.gu.anghammarad.testutils.TryValues
+import org.scalatest.EitherValues
 
-import scala.collection.JavaConverters._
+import scala.jdk.CollectionConverters.SeqHasAsJava
 import scala.io.Source
 
-class SerializationTest extends FreeSpec with Matchers with EitherValues with TryValues {
+
+class SerializationTest extends AnyFreeSpec with Matchers with EitherValues with TryValues {
 
   "Notification Serialization" - {
     val validJsonString = Source.fromURL(getClass.getResource("/notification.json")).mkString
-    val validJson = parse(validJsonString).right.value
+    val validJson = parse(validJsonString).value
```

Comment:
I'm surprised this works! I thought the `.value` from scalatest's `EitherValues` trait was still generating `warnings` in Scala 2.13, which *should* be a compile error if things are set up properly. I'm assuming that's not the case! If this compiles and the tests still work then nothing to worry about but I would double check that tests are being compiled and run at the moment? It might just be that we don't have `fatal-warnings` enabled on this project yet. If this is working now then no worries, and we've just got some tech debt to enable that and fix the warnings in the future.

If you do find these are broken holler at me and I'll shoot you the helper I use to make this work, hopefully not necessary!

Reply:
Without this change the tests were throwing warnings. Apparently this is right-biased in later versions of circe. Tests all pass happily without throwing those warnings when the .right is removed.